bob/bob.py

```diff
@@ -1,2 +1,10 @@
 def hey(phrase):
-    pass
+    if phrase.isspace() or not phrase:
```

Comment:
I'd start with "sanitizing" `phrase` as the very first thing in this function. Then all the subsequent checks will be shorter:
```
# 'strip' removes all white-space characters
phrase_stripped = phrase.strip()
if not phrase_stripped:
    return 'Fine. Be that way!'
```
And then use phrase_stripped from now on.